### ISSUE TITLE
Is there any way to parse the layout json and get a 2D interior layout?

### ISSUE THREAD

**Issue by universewill** [3D-FRONT-FUTURE/3D-FRONT-ToolBox#11]
I want to get 2d layout of interiors, how to do that?

**Comment by jwzxgy2007** [comment]
Hi, universewill.

The 2d layout you mean is one of the following images or just 2d position information of interiors?
<img src="https://user-images.githubusercontent.com/14820455/127621996-c295964e-16fb-4b0d-9857-5e477b4373a3.png" width = "200" align=center /><img src="https://user-images.githubusercontent.com/14820455/127622687-68fb0fa1-5035-4481-8834-bdad4d07f917.png" width = "200" align=center />

**Comment by universewill** [comment]
> Hi, universewill.
> 
> The 2d layout you mean is one of the following images or just 2d position information of interiors?
> <img alt="" width="200" src="https://user-images.githubusercontent.com/14820455/127621996-c295964e-16fb-4b0d-9857-5e477b4373a3.png"><img alt="" width="200" src="https://user-images.githubusercontent.com/14820455/127622687-68fb0fa1-5035-4481-8834-bdad4d07f917.png">

2d position and bouding box or contour infomation of interiors, which contains wall, window, door, wardrobe, bed, sofa and other funitures in the designed interior

**Comment by fathyshalaby** [comment]
I would be interested in the 2d position information. Could you explain that.

**Comment by universewill** [comment]
> Hi, universewill.
> 
> The 2d layout you mean is one of the following images or just 2d position information of interiors? <img alt="" width="200" src="https://user-images.githubusercontent.com/14820455/127621996-c295964e-16fb-4b0d-9857-5e477b4373a3.png"><img alt="" width="200" src="https://user-images.githubusercontent.com/14820455/127622687-68fb0fa1-5035-4481-8834-bdad4d07f917.png">

yes. is there any way to generate the 2d layout including doors, windows, walls and other furnitures?
